addons/account_payment/controllers/portal.py

```diff
@@ -72,3 +87,14 @@ def _invoice_get_page_view_values(self, invoice, access_token, **kwargs):
         new_values = portal_page_values | payment_form_values | payment_context | self._get_extra_payment_form_values(**kwargs)
         values |= new_values
         return values
+
+    @http.route()
+    def portal_my_invoice_detail(self, invoice_id, access_token=None, report_type=None, download=False,
+                                 payment=False, amount=None, payment_token=None, **kw):
```

Comment:
You can just remove the unnecessary kwargs from the method signature and they will be packed in `kw`.

Reply:
Not a big deal but it could just have been:
`def portal_my_invoice_detail(self, invoice_id, payment_token=None, **kw):`

Reply:
Updated